content/en/docs/setup/install/multicluster/single-network/initial-configuration/index.md

```diff
@@ -0,0 +1,205 @@
+---
+title: Initial Configuration
+description: Describes how to complete the initial configuration for a multicluster mesh on a flat network.
+weight: 10
+keywords: [multicluster,first-cluster]
+owner: istio/wg-environments-maintainers
+test: n/a
+---
+
+To build a {{< gloss >}}multicluster{{< /gloss >}} deployment, configure a
+single cluster {{< gloss >}}service mesh{{< /gloss >}} as your starting point.
+
+The single cluster mesh you deploy in this section, serves as a model for the
+configuration of the first {{< gloss >}}primary cluster{{< /gloss >}}. You need
+at least one primary cluster to complete the [different multicluster deployments.](/docs/ops/deployment/deployment-models/#multiple-clusters)
+
+Each cluster in a multicluster deployment needs a unique identifier. Using
+`CLUSTER_1` as a unique identifier for the first cluster, the following
+diagram shows a service mesh spanning a single primary cluster.
+
+{{< image width="75%"
+    link="init-conf.svg"
+    caption="A service mesh with a single primary cluster"
+    >}}
```

Comment:
The bottom box lists "Proxy" and a box with the Envoy logo as if they are different things.

Reply:
All proxies in all diagrams are depicted that way, so I'm keeping things consistently.